sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobHierarchyListSegment.java

```diff
@@ -6,17 +6,14 @@
 
 import com.azure.core.implementation.annotation.Fluent;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
-
 import java.util.ArrayList;
 import java.util.List;
 
 /**
  * The BlobHierarchyListSegment model.
  */
 @JacksonXmlRootElement(localName = "Blobs")
-@JsonDeserialize(using = CustomHierarchicalListingDeserializer.class)
```

Comment:
This is the only change in the first commit that wasn't just moving things around and I'm not sure what in the swagger changed to stop generating it.

Reply:
This was added back manually last I checked, @jianghaolu is that correct?

Reply:
@alzimmermsft correct. This has never been generated; it's always been added manually. We have to add it back every time we generate. I don't know if I could find it, but I'm pretty sure I filed an issue against autorest.java to try to get the deserialization fixed a while back

Reply:
Okay - thanks.  I feel less crazy now.  😄 

@alzimmermsft - can you both add this back and then add a note in the README.md for Blobs to add this back in after regenerating?  We might be able to address this with a regex at some point, but I don't want to block on getting that working now.

Reply:
Added it back and added a comment in the README about this issue.

Reply:
This is because Storage is returning something like

```xml
<BlobItems>
<BlobPrefix>prefix1</BlobPrefix>
<Blob>blob1</Blob>
<BlobPrefix>prefix2</BlobPrefix>
<Blob>blob2</Blob>
</BlobItems>
```

And without this annotation blob prefix list will only have `prefix2` and blob list will only have `blob2`. Assuming this XML format is correct, I'm wondering if there's a way to represent this in Swagger. If we can represent it in Swagger, then I can conditionally generate this annotation in Java code.